README.md

```diff
@@ -935,6 +935,7 @@ Community Repos:
 * [Netflix/bless :fire::fire::fire::fire::fire:](https://github.com/Netflix/bless) - SSH Certificate Authority that runs as a Lambda function.
 * [Netflix/security_monkey :fire::fire::fire::fire::fire:](https://github.com/Netflix/security_monkey) - Monitors policy changes and alerts on insecure configurations.
 * [RiotGames/cloud-inquisitor :fire::fire:](https://github.com/RiotGames/cloud-inquisitor) - Tool to enforce ownership and data security.
+* [salesforce/policy_sentry :fire::fire::fire:] - Generate secure, least privilege IAM Policies automagically  
```

Comment:
@kmcquade could you add the link here and refresh the description based on current repo description?

* [salesforce/policy_sentry :fire::fire::fire:](https://github.com/salesforce/policy_sentry) - IAM Least Privilege Policy Generator.

Reply:
@donnemartin for some reason I'm not able to edit this PR. I think I must have deleted my fork. I'm going to close this one and then make a new one. Cool?

Reply:
Sounds good!